content/intro-to-storybook/vue/es/deploy.md

```diff
@@ -1,100 +1,138 @@
 ---
 title: 'Desplegar Storybook'
 tocTitle: 'Desplegar'
-description: 'Desplegar Storybook online con GitHub y Netlify'
+description: 'Aprenda a desplegar Storybook en línea'
 commit: '107b7ce'
 ---
 
-En este tutorial hemos ejecutado Storybook en nuestra máquina de desarrollo. También se puede compartir ese Storybook con el equipo, especialmente con los miembros no técnicos. Afortunadamente, es fácil implementar Storybook en línea.
-
-<div class="aside">
-<strong>¿Hiciste los test con Chromatic antes?</strong>
-<br/>
-🎉 Sus historias ya están desplegadas! Chromatic indexa de forma segura sus historias en línea y las rastrea a través de ramas y commits. Salta este capítulo y ve a la <a href="/intro-to-storybook/vue/es/conclusion">conclusión</a>.
-</div>
+A lo largo de este tutorial, creamos componentes en nuestra máquina de desarrollo. En algún momento, tendremos que compartir nuestro trabajo para recibir comentarios del equipo. Despleguemos Storybook en línea para ayudar a los compañeros de equipo a revisar la implementación de la interfaz de usuario.
 
 ## Exportando como una app estática
 
 Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook, solo tenemos que cambiarla como lo hicimos antes cuando el proyecto se inicializó en la [sección de Empezando](/intro-to-storybook/vue/es/get-started).
 
```

Comment:
@IvanSotelo if you could adjust this to :

```markdown
Para desplegar Storybook primero necesitamos exportarlo como una aplicación web estática. Esta funcionalidad ya está incorporada en Storybook.
```

It generated a redundancy as we're mentioning the same chapter below